addons/OPT/GPS/fn_miniGPS.sqf

```diff
@@ -24,12 +24,21 @@
 
 #include "macros.hpp"
 
-GVAR(MiniMapCount) = 0;
-GVAR(MiniMapCountOld) = 0;
+//Init Titel für setzen uiNamespace "opt_ui_gps_minimap"
+uiNamespace setVariable ['opt_ui_gps_minimap', controlNull];
 
-//Init Titel für setzen uiNamespace
-titleRsc ["opt_gps_minimap","plain"];
-titleFadeOut 0;
+//Kippstufe für ein/ausblenden der Mini GPS Karte
+DFUNC(Kippstufe) = 
```

Comment:
`Kippstufe` ist ein bisschen generisch. Um den Code so leserlich wie möglich zu machen, würde ich diese Funktion `toggleMiniGPS` nennen :)

Reply:
Test test

Reply:
Das ist genaus generisch wie "Kippstufe". Was hast du denn gegen "toggleMiniGPS"?

Reply:
Umschalten heißt toggle. Und ich bin für Deutsche begriffe.

Reply:
also `kippstufeMiniGPS`

Reply:
ihr nervt gerne :-), dann heist das UmschaltungMiniGPS und dann ist aber auch gut !

Reply:
> ihr nervt gerne :-)

Es geht darum den Code leserlich zu machen. Und ja für dieses Ziel nerve ich gerne :P

> Umschalten heißt toggle. Und ich bin für Deutsche begriffe.

Ja und nein. Das Englische "toggle" hat (insbesondere in diesem Zusammenhang) noch eine Bedeutung, die sich meines Erachtens nach nur mühselig ins Deutsche übersetzen lässt. https://www.dict.cc/?s=toggle listet "hin- und herschalten" was genau das ist, was in diesem Kontext passiert. Dh wenn man das Ganze deutsch machen wollte, müsste man das eigentlich "MiniGPSHinUndHerSchalten" nennen. Das ist aber super umständlich.
Deutsche Begriffe im IT-Bereich sind nicht sinnvoll. Aber diese Diskussion hatten wir ja bereits mal an anderer Stelle. Wenn du beim Programmieren lieber deutsche Begriffe verwendest, dann von mir aus, aber wenn der Namensvorschlag eingereicht wird und du das nicht selbst übersetzen musst, finde ich es nicht sinnvoll sich da aufs Deutsche zu verstarren. Jeder der ein bisschen Programmiererfahrung hat, weiß was `toggle` bedeutet, wohingegen die Wenigstens auf Anhieb verstehen werden, dass `Kippschalter` oder `Umschalter` / `Umschaltung` das selbe sein soll.

Davon abgesehen präferieren in diesem Team glaube ich alle außer dir englische Begriffe und wenn einer von denen den Code mal lesen will/muss, dann fängt diese Person aus oben genannten Gründen mit `toggle` mehr an. Als Urheber des Codes weißt du ohnehin, was die Funktion macht, dh. da ist die Nomenklatur zweitrangig.

Auf dieser Grundlage spricht meiner Meinung nach deutlich mehr für eine englische Nomenklatur als dagegen...

Reply:
Jetzt ist aber mal gut bitte, über der Funktion stehen genau das die macht. 
Jeder der Umschaltung nicht verstehen sollte kann das danach vorziehen. 

Und ihr könnt ja von mir aus alle in Japanisch schreiben, Ich habe dazu schon mal gesagt das das Träumereien von euch sind das hier scharen von Englischen Programmiert mal auf tauchen werden. 
Hier werden immer nur Deutsche Leute daran arbeiten und wer neuen Leuten nach einigen Jahren das verstehen des Codes erschweren will kann das auf Englisch machen.
Ich werde das nicht tun! 
Ich bin sehr Froh das James, nicht überall gemacht hat und so habe ich eine Chance da seine Gedanken zu verstehen. Ich habe da null Bock die Englischen Kommentare zu übersetzen, damit ich nach voll ziehen kann was da vorhat. 
Wir reden deutsch also können wir auch deutsch Schreiben.

Reply:
Wie gesagt: Mach dein Zeug auf deutsch. Schreib deine Kommentare auf Deutsch. Ist okay, da du Englisch halt nicht so gut kannst. Kein Problem.
Aber wenn ich dir sage, dass der englische Name hier besser passt, weil er einfach **viel** besser beschreibt,was passiert, dann gibt es keinen Grund sich da mit Händen und Füßen dagegen zu wehren.

Ich will ja nicht dass du all deinen Code übersetzt oder sowas. Ich will nur dass dieser Funktionsname, der einfach nicht vernünftig ins Deutsche übersetzt werden kann, auf Englisch benannt wird. Da musst du nur copy&paste machen

Reply:
weil dir so am Herzen liegt ändere ich dann den Namen :-)

Reply:
Danke :heart: